api-types.ts

```diff
@@ -9,25 +9,77 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
+  Date: any;
 };
 
+
 export type Group = {
   __typename?: 'Group';
   id?: Maybe<Scalars['Int']>;
   name?: Maybe<Scalars['String']>;
 };
 
+export type GroupInput = {
+  name?: Maybe<Scalars['String']>;
+};
+
 export type Query = {
   __typename?: 'Query';
   listGroups: Array<Maybe<Group>>;
+  listShipments: Array<Maybe<Shipment>>;
```

Comment:
Why return a list of maybe shipments? Here a list of 0 or more Shipments is returned. Why would you return a list of items that can be a shipment or an absent shipment?

Reply:
This will be updated when I address your other comment to make `listShipments` return `[Shipment!]!`.